lib/sycamore/sycamore/llms/guidance.py

```diff
@@ -0,0 +1,37 @@
+from sycamore.llms.prompts.default_prompts import SimplePrompt
+from guidance.models import Chat, Instruct, Model as GuidanceModel
+from guidance import gen, user, system, assistant, instruction
+
+
+def _execute_chat(prompt: SimplePrompt, model: GuidanceModel, **kwargs) -> str:
+    with system():
+        lm = model + prompt.system.format(**kwargs)
+
+    with user():
+        lm += prompt.user.format(**kwargs)
+
+    with assistant():
+        lm += gen(name=prompt.var_name)
+
+    return lm[prompt.var_name]
```

Comment:
At this point `var_name` is entirely superfluous, right? I think it's a vestige of the time when we returned the dict, but now since we return the string, it's entirely wrapped in the the internal LLM call and never exposed? I don't think you need to remove it in this PR, but we probably could, right?

Reply:
I think that makes sense? Tbh I don't remember how we got here